aws/table_aws_vpc_verified_access_endpoint.go

```diff
@@ -0,0 +1,281 @@
+				Description: "The ID of the AWS verified access endpoint.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "verified_access_group_id",
+				Description: "The ID of the AWS verified access group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "verified_access_instance_id",
+				Description: "The ID of the AWS verified access instance.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_time",
+				Description: "The creation time.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "status_code",
+				Description: "The endpoint status code. Possible values are pending, active, updating, deleting or deleted.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Status.Code"),
+			},
+			{
+				Name:        "application_domain",
+				Description: "The DNS name for users to reach your application.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "attachment_type",
+				Description: "The type of attachment used to provide connectivity between the AWS verified access endpoint and the application.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "deletion_time",
+				Description: "The deletion time.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "A description for the AWS verified access endpoint.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "device_validation_domain",
+				Description: "Returned if endpoint has a device trust provider attached.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "domain_certificate_arn",
+				Description: "The ARN of a public TLS/SSL certificate imported into or created with ACM.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "endpoint_domain",
+				Description: "A DNS name that is generated for the endpoint..",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "endpoint_type",
+				Description: "The type of AWS verified access endpoint. Incoming application requests will be sent to an IP address, load balancer or a network interface depending on the endpoint type specified. Possible values are load-balancer or network-interface.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "last_updated_time",
+				Description: "The last updated time.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "load_balancer_options",
+				Description: "The load balancer details if creating the AWS verified access endpoint as load-balancertype.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network_interface_options",
+				Description: "The options for network-interface type endpoint.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "status",
```

Comment:
Another attribute `Message` is available inside status field. The status message of the Verified Access endpoint.
Do you think it will be add value if we will add

Reply:
don't think adding `status_message` would be beneficial. `status_code` would suffice 👍.